src/authz-smart/smartConfig.ts

```diff
@@ -2,6 +2,35 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
+import { TypeOperation, SystemOperation } from 'fhir-works-on-aws-interface';
+

```

Comment:
Normally we have our Config interfaces in `fhir-works-on-aws-interface`. For example `FHIRConfig` and `Auth`. What are your thoughts on putting it in the `interface` package vs putting it here?

Reply:
Yeah so this will be much like how we set up RBAC (https://github.com/awslabs/fhir-works-on-aws-authz-rbac/blob/mainline/src/RBACConfig.ts)

It is a configuration that doesn't apply widely just to the SMART implementation